setup.cfg

```diff
@@ -27,8 +27,8 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    audbackend >=0.3.17, <1.0.0
-    audeer >=1.20.0
+    audbackend @ git+https://github.com/audeering/audbackend.git@artifactory-old-structure
```

Comment:
Before we merge, we should release a new version of audbackend and change to `audbackend >=1.0.0`.

Reply:
I rebased `main` which introduces `pyproject.toml` and I updated the dependency there to `audbackend >=1.0.0`